models/Petition.model.js

```diff
@@ -3,10 +3,29 @@ const Schema = mongoose.Schema;
 
 const petitionSchema = Schema(
   {
+    userId: Number,
```

Comment:
Hey @cqnguy23 I carried over a SQL pattern here,

I think this would best be

```js
owner: {
    ref: "User",
    required: true,
    type: Schema.Types.ObjectId,
  },
```

If we do it this way, we can do `petition.owner` and more easily find the user.